### ISSUE TITLE
Monitor resource usage and adjust pipeline interface accordingly

### ISSUE THREAD

**Issue by vreuter** [databio/dnameth_pipelines#16]
Now may be a good time to assess resource usage of the bisulfite pipelines, when a variety of samples are being processed in relative proximity. The pipeline interface could be updated to reflect observation of resource usage during these runs.

**Comment by nsheff** [comment]
good idea. make sure you're using the latest pypiper, which increased accuracy for mem monitoring.

**Comment by nsheff** [comment]
also don't make the bounds too tight... nothing worse than getting a long-running job killed because it barely went over some limit.
